packages/esm-patient-orders-app/src/components/orders-details-table.component.tsx

```diff
@@ -389,6 +400,24 @@ function InfoTooltip({ orderer }: { orderer: string }) {
   );
 }
 
+function ExpandedOrderView({ row }: { row: any }) {
+  const { t } = useTranslation();
+  let orderActions = row.cells.find((cell) => cell.info.header === 'actions');
+  let orderItem = orderActions.value?.props?.orderItem;
+
+  if (orderItem.type == 'drugorder') {
+    return <MedicationRecord medication={orderItem} />;
+  } else if (orderItem.type == 'testorder') {
+    return <TestOrder testOrder={orderItem} />;
+  } else {
+    return (
+      <div>
+        <p>{t('unknownOrderType', 'Unknown Order Type')}</p>
```

Comment:
What kind of tests fall under unknown?

Reply:
This is widget shows all orders so i added that option to future proof it if another order type is added in the future.